schemas/metaschema/DatabaseCategory.schema

```diff
@@ -0,0 +1,4 @@
+{
+  Id: { domain: 'Id', required: true },
+  Symbol('allowNotDefinedFields'): false
```

Comment:
Perhaps this was meant to be `Symbol.for`? As `Symbol` will make it impossible to actually get this field IIUC.
Also, I'd suggest to 'namespace' it `@metarhia/allowNotDefinedFields`.